network-api/networkapi/wagtailpages/pagemodels/research_hub/landing_page.py

```diff
@@ -11,8 +20,35 @@ class ResearchLandingPage(foundation_metadata.FoundationMetadataPageMixin, wagta
         'ResearchAuthorsIndexPage',
     ]
 
+    intro = models.CharField(
+        blank=True,
+        max_length=250,
+    )
+
+    content_panels = [
+        FieldPanel(
+            'title',
+            classname='full title',
+            widget=TitleWidget(attrs={"class": "max-length-warning", "data-max-length": 60})
+        ),
+        FieldPanel('intro'),
+        InlinePanel('featured_topics', heading="Featured Topics"),
+    ]
+
     def get_context(self, request):
         context = super().get_context(request)
         ResearchLibraryPage = apps.get_model("wagtailpages", "ResearchLibraryPage")
         context['library_page'] = ResearchLibraryPage.objects.first()
+        context['latest_research_detail_pages'] = self.get_recent_research_pages
```

Comment:
**Suggestion:** For consistency, I think it would be nice if we stick to either "latest" or "recent", but not mix them. E.g. use `get_latest_research_detail_pages` for the method to match the context variable.

Reply:
Updated!